test/testenv/buildx.go

```diff
@@ -59,59 +58,48 @@ func (b *BuildxEnv) Load(ctx context.Context, id string, f gwclient.BuildFunc) e
 	return nil
 }
 
-func (b *BuildxEnv) version(ctx context.Context) (string, error) {
-	cmd := exec.CommandContext(ctx, "docker", "buildx", "version")
+func (b *BuildxEnv) supportsDialStdio(ctx context.Context) (bool, error) {
+	cmd := exec.CommandContext(ctx, "docker", "buildx", "--help")
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		return "", pkgerrors.Wrap(err, string(out))
+		return false, pkgerrors.Wrap(err, string(out))
 	}
 
-	fields := strings.Fields(string(out))
-
-	if len(fields) != 3 {
-		return "", errors.New("could not determine buildx version")
+	if !strings.Contains(string(out), "dial-stdio") {
```

Comment:
Maybe add a comment about the logic here:
`buildx --help` includes `dial-stdio` => `dial-stdio` supported

Reply:
Sure, updated.
Also cleaned up the logic a little bit after reading more of the code.